pkg/systemstatsmonitor/system_stats_monitor.go

```diff
@@ -89,6 +90,9 @@ func NewSystemStatsMonitorOrDie(configPath string) types.Monitor {
 		ssm.memoryCollector = NewMemoryCollectorOrDie(&ssm.config.MemoryConfig)
 	}
 	if len(ssm.config.OsFeatureConfig.MetricsConfigs) > 0 {
+		// update the known_modules_path to relative the system-stats-monitors path
+		ssm.config.OsFeatureConfig.KnownModulesConfigPath = filepath.Join(filepath.Dir(configPath),
```

Comment:
can the test be added here: https://github.com/kubernetes/node-problem-detector/blob/master/pkg/systemstatsmonitor/system_stats_monitor_test.go?